api-get-object-file.go

```diff
@@ -78,8 +78,13 @@ func (c Client) FGetObject(bucketName, objectName, filePath string) error {
 		return err
 	}
 
+	// Initialize get object conditions and set the appropriate
+	// range offsets to read from.
+	reqHeaders := NewGetReqHeaders()
+	reqHeaders.SetRange(st.Size(), 0)
```

Comment:
@krisis it looks confusing but st.Size() is actually size of the partially downloaded file, not the size of the object